src/utils/axios/axiosInstance.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const instance = axios.create({
```

Comment:
I would recommend more flexible structure  future use cases
i do not have something  specific in mind currently but we can think of something like 
```javascript
const base_url = "localhost/5000"

export const api = (url, data, method, params) => { // any we can keep updating by adding extra params e.g. params, cancel or user tokens ..etc
    const request = axios({
        method,
        url: base_url + url,
        data: JSON.stringify(data),
        params,
        headers: {
            'Content-Type': 'application/json',
            'Accept: 'application/json',
            'Access-Control-Allow-Origin': '*',
        }
    })

    return request
}
```